charts/ccsm-helm/test/identity/golden/deployment.golden.yaml

```diff
@@ -0,0 +1,65 @@
+---
+# Source: ccsm-helm/charts/identity/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ccsm-helm-test-identity
+  labels:
+    app: camunda-cloud-self-managed
+    app.kubernetes.io/name: identity
+    app.kubernetes.io/instance: ccsm-helm-test
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/part-of: camunda-cloud-self-managed
+    app.kubernetes.io/version: "SNAPSHOT"
+    app.kubernetes.io/component: identity
+  annotations:
+    {}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: camunda-cloud-self-managed
+      app.kubernetes.io/name: identity
+      app.kubernetes.io/instance: ccsm-helm-test
+      app.kubernetes.io/managed-by: Helm
+      app.kubernetes.io/part-of: camunda-cloud-self-managed
+      app.kubernetes.io/component: identity
+  template:
+    metadata:
+      labels:
+        app: camunda-cloud-self-managed
+        app.kubernetes.io/name: identity
+        app.kubernetes.io/instance: ccsm-helm-test
+        app.kubernetes.io/managed-by: Helm
+        app.kubernetes.io/part-of: camunda-cloud-self-managed
+        app.kubernetes.io/version: "SNAPSHOT"
+        app.kubernetes.io/component: identity
+    spec:
+      containers:
+      - name: identity
+        image: "camunda/identity:SNAPSHOT"
+        env:
+          - name: SERVER_PORT
+            value: "8080"
```

Comment:
This can be removed as `8080` is the default (if this configuration is not needed for something else)